python_ta/cfg/graph.py

```diff
@@ -3,12 +3,14 @@
 from typing import Any, Dict, Generator, List, Optional, Set
 
 try:
+    import z3
```

Comment:
We should use a consistent import style here. Either import `Solver` on the line below, or only import `z3` and then modify the usages of `Z3_OP_UNINTERPRETED` to `z3.Z3_OP_UNINTERPRETED`, etc. consistently throughout.